__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -9,6 +9,13 @@ import ChatAdapterProtocols from "../src/core/ChatAdapterProtocols";
 import AriaTelemetry from "../src/telemetry/AriaTelemetry";
 
 describe('Omnichannel Chat SDK', () => {
+    // beforeEach(() => {
```

Comment:
Can you please remove the commented line? If not, please add comment why do we need it?

Reply:
Forgot to remove it 😅 Done!